datafusion/proto/proto/datafusion.proto

```diff
@@ -481,7 +481,8 @@ message ScalarUDFExprNode {
 }
 
 enum BuiltInWindowFunction {
-  ROW_NUMBER = 0;
+  UNSPECIFIED = 0; // https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum
+  // ROW_NUMBER = 0;
```

Comment:
The proto enum requires the first value to be zero. See: https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum

Leaving the older value commented out instead of removing it so we know that it was `ROW_NUMBER`. This will anyways be removed once all the other built-ins are moved.